CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+Guidelines for contributing to Hibernate Search
+====
+Contributions from the community are essential in keeping Hibernate Search strong and successful.
+This guide focuses on how to contribute back to Hibernate Search using GitHub pull requests.
+If you need help with cloning, compiling or setting the project up in an IDE please refer to
+[this wiki document](https://developer.jboss.org/wiki/ContributingtoHibernateSearch).
```

Comment:
I'd merge simply point to the website (instead of the wiki) so we can merge this PR. Improving the website can be done later. I'll put a banner on the old wiki now.